src-vue/src/plugins/store.ts

```diff
@@ -61,7 +61,7 @@ export const store = createStore<FlightCoreStore>({
     state(): FlightCoreStore {
         return {
             developer_mode: false,
-            game_install: {} as unknown as GameInstall,
+            game_install: {game_path: "", profile: "", install_type: "UNKNOWN"} as GameInstall,
```

Comment:
Can we set `game_path` as `undefined` or something. Setting to `""` feels kinda dangerous. Imagine we modify code to at some point run

```rust
let folder_to_delete = format!("{}/", game_install.game_path);
delete_folder_recursively(folder_to_delete);
```

We really don't wanna [pull a Valve](https://github.com/ValveSoftware/steam-for-linux/issues/3671#issuecomment-70161790) ;)